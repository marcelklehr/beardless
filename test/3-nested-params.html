<div><a href="" data-template="nested.link"></a><a href="" data-template="nested.unlink"></a></div>