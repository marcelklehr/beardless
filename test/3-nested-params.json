{"nested":
  { "link": "Hello world"
  , "unlink": false
  }
}