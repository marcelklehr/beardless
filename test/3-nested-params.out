<div><a href="">Hello world</a></div>